pkg/netpol/connlist/conns_formatter_txt.go

```diff
@@ -60,6 +60,9 @@ func (t *formatText) writeExposureOutput(exposureResults []ExposedPeer) string {
 	sort.Strings(unprotectedLines)
 	// writing results of exposure for all peers
 	res := exposureAnalysisHeader
+	if len(egressExpLines) == 0 && len(ingressExpLines) == 0 {
+		return res + newLineChar
```

Comment:
shouldn't `exposureAnalysisHeader` be always with newline already??

Reply:
true, had the new line char at the beginning of the new sub-section; fixed